docs/quickstart.rst

```diff
@@ -534,28 +538,25 @@ Now you should verify that the topics are created before moving on:
 
   .. sourcecode:: bash
 
-    docker run \
+    $ docker run \
        --net=host \
        --rm \
        confluentinc/cp-kafka:3.3.0-SNAPSHOT \
        kafka-topics --describe --zookeeper localhost:32181
 
-For this example, we'll create a FileSourceConnector, a FileSinkConnector and directories for storing the input and output files. If you are running Docker Machine then you will need to SSH into the VM to run these commands by running ``docker-machine ssh <your machine name>``.
+For this example, we'll create a FileSourceConnector, a FileSinkConnector and directories for storing the input and output files.
 
-  First, let's create the directory where we'll store the input and output data files.  Remember, we must do this within the Docker Host.
+  First, let's create the directory where we'll store the input and output data files.
```

Comment:
Since this is the first time we're running `docker exec` (?), perhaps it's worth adding something like "in the running container."